### ISSUE TITLE
Add hacktoberfest hash tag to tweets

### ISSUE THREAD

**Issue by Roshanjossey** [firstcontributions/firstcontributions.github.io#13]
Adding hacktoberfest tag in our tweets will give it more visibility.

🎯 **Goal**
Reach out to more people who follows hacktoberfest tag on twitter.

💡 **Possible solutions**
Change twitter share link https://github.com/firstcontributions/firstcontributions.github.io/blob/source/src/components/SocialShare/Twitter.jsx#L4

from 

```
const twitterLink = "https://twitter.com/intent/tweet?text=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20@1stcontribution.%20You%20can%20too%20at%20https%3A//goo.gl/66Axwe%0A&hashtags=OpenSource,CodeNewbie"
```

to

```
const twitterLink = "https://twitter.com/intent/tweet?text=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20@1stcontribution.%20You%20can%20too%20at%20https%3A//goo.gl/66Axwe%0A&hashtags=OpenSource,CodeNewbie,Hacktoberfest"
```

📋  **Steps to solve the problem**

*   Comment below about what you've started working on.
*   Add, commit, push your changes.
*   Submit a pull request and add this in comments - `Addresses #<put issue number here>`
*   Ask for a review in comments section of pull request.
*   Celebrate your contribution to this project. 🎉

**Comment by Darkpingouin** [comment]
I'm working on it :)

**Comment by Roshanjossey** [comment]
@Darkpingouin, do you know if there is something similar to tags in facebook and linkedin share?

**Comment by Darkpingouin** [comment]
@Roshanjossey I have no idea but I could look it up and add Hacktoberfest on them like I did on my pull request for Twitter

**Comment by Roshanjossey** [comment]
@Darkpingouin, cool. Please create issues for them as well.

**Comment by sara-02** [comment]
Fixed in #14
